### PR TITLE
Documentation Fixes - 2019b

### LINKED ISSUE #7451: Documentation Issue 2019b
Issue overview
--------------
A collection of minor documentation issues. See comments below for additional items.

- [x] I/O Ref "Zone Equipment Input Syntax Map" figure is missing.
- [x] I/O Ref "Representative Zone Equipment" figure is missing (next the previous one)

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (list location of defect file here): N/A
 - [x] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by jmarrec**
Pull request overview
---------------------
 - Fixes #7451
 - Fixes various documentation issues, see changelog that summarizes the items in the original issue

### Changelog:

- [x] I/O Ref "Zone Equipment Input Syntax Map" figure is missing.
- [x] I/O Ref "Representative Zone Equipment" figure is missing (next the previous one)
- [x] Fan:SystemModel is missing as a valid fan type in FanPerformance:NightVentilation object in I/O reference documentation [here](https://github.com/NREL/EnergyPlus/blob/develop/doc/input-output-reference/src/overview/group-fans.tex#L711-L713).
- [x] Broken link on page 2639 https://www.energy.ca.gov/title24/2005standards/archive/rulemaking/documents/tdv/index.html: EnvironmentalImpactFactors
- [x] Typo "crwon" in IO Ref 9.1 WindowMaterial:Blind:EquivalentLayer
- [x] There are a bunch of old EnergyPlus links (eg apps1.eere.energy.gov/buildings/energyplus/cfm/weather_data.cfm) in the Tips & Tricks documentation, perhaps others.
    * **https://energyplus.net/features is out of date**, trying to replace https://github.com/NREL/EnergyPlus/blob/v9.2.0/doc/tips-and-tricks-using-energyplus/src/general/comparing-energyplus-to-other-programs.tex#L11
- [x] IORef [Figure 1.111](https://github.com/NREL/EnergyPlus/blob/fd07e7599f8b921178db9ced3f96aba53c39d582/doc/input-output-reference/src/overview/group-air-distribution.tex#L723) is missing a caption: AirLoopHVAC:OutdoorAirSystem:EquipmentList
- [x] Some figures do not appear in the docs (on mac, at least). For example, [Figure 1.110](https://github.com/NREL/EnergyPlus/blob/fd07e7599f8b921178db9ced3f96aba53c39d582/doc/input-output-reference/src/overview/group-air-distribution.tex#L555).
    * I realized that this was due to these being 16-bit images, so I converted them to 8-bit color
- [x] The [example IDF object for Output:Table:Annual](https://bigladdersoftware.com/epx/docs/9-1/input-output-reference/output-table-annual.html#outputtableannual) has an extra field "Report Group Name" that isn't in the IDD.
- [x] Input Output Reference for CoolingTower:VariableSpeed:Merkel does not mention these input fields but they are in the idd file:
    ```
    N17, \field Design Inlet Air Dry-Bulb Temperature
    N18, \field Design Inlet Air Wet-Bulb Temperature
    N19, \field Design Approach Temperature
    N20, \field Design Range Temperature
    ```
- [x] Same for CoolingTower:SingleSpeed: Already done via https://github.com/NREL/EnergyPlus/commit/c3840d1583a2880f5bf631e8315de7098a99b427, but updated the example IDF snippets (and used actual units via sixunits instead of using \circ etc)
- [x] Same for CoolingTower:TwoSpeed
- [x] Input Output Reference discussion of CoolingTower:VariableSpeed:Merkel is missing value inside quotes in discussion of 1.24.4.1.4 Field: Performance Input Method.  “” should be replaced with “UFactorTimesAreaAndDesignWaterFlowRate” in the second sentence below.
- [x] I/O reference groups all 'Outputs' at the bottom of the 'Group - Room Air Models' section, not after the relevant object input fields.
- [x] The headings of this table seem messed up in the IOref 9.2: Table 1.21 Metabolic Rates for Various Activities
- [x] In the Controller:MechanicalVentilation object documentation and ~IDF Editor~ IDD, change field `Zone <x> Name` to `Zone or ZoneList <x> Name` as this field does accept a ZoneList object.
- [x] The [Field: Solar Distribution](https://bigladdersoftware.com/epx/docs/9-2/input-output-reference/group-simulation-parameters.html#field-solar-distribution) says for FullExterior:
"... transmitted diffuse radiation, which is assumed to be uniformly distributed on all interior surfaces."
However, the [Surface Inside Face Initial Transmitted Diffuse Absorbed Solar Radiation Rate](https://bigladdersoftware.com/epx/docs/9-2/input-output-reference/group-thermal-zone-description-geometry.html#surface-inside-face-initial-transmitted-diffuse-absorbed-solar-radiation-rate-w) output suggests this is no longer correct as it states:
"As of Version 2.1, diffuse solar transmitted through exterior and interior windows is no longer uniformly distributed. Instead, it is distributed according to the approximate view factors between the transmitting window and all other heat transfer surfaces in the zone."

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by jmarrec** [comment]
**NOTE: Last commit 6992625 introduces an IDD change**

**Comment by Myoldmopar** [comment]
There is still one item marked as TODO.  I am ready to review the changes on this PR but I will hold off if that is still going to be addressed.

**Comment by jmarrec** [comment]
@Myoldmopar i addressed the TODO but asked confirmation from @mjwitte to ensure i understood his directives correctly (cf review). You can review this PR including that part

**Comment by jmarrec** [comment]
@mjwitte Thanks, I missed that one. Addressed in https://github.com/NREL/EnergyPlus/pull/7666/commits/a9e4657bc043ad95f049cf87fcf4c78a64338534

**Comment by Myoldmopar** [comment]
Great, building locally now to check it out.

**Comment by jmarrec** [comment]
Fix #7618 too:

image

File: doc/input-output-reference/src/overview/group-condenser-equipment.tex [hunk 1]
```diff
@@ -115,19 +115,23 @@ \subsubsection{Inputs}\label{inputs-007}
 
 \paragraph{Field: Design Inlet Air Dry-Bulb Temperature}\label{field-design-inlet-air-drybulb-temperature}
 
-This numeric field specifies the design inlet air dry-bulb temperature of the tower. The default is 35.0\(*^{\circ}\)C.
+This numeric field specifies the design inlet air dry-bulb temperature of the tower. The default is \SI{35.0}{\celsius}.
```
Comment (mitchute) [inline doc/input-output-reference/src/overview/group-condenser-equipment.tex:118]: This right here 👍 👍 👍 

We need to (eventually) clean up the units throughout all docs with ```\SI```.

**Review by mitchute — APPROVED** [review]
This is ready to merge. I'll pull in develop one last time, then merge when CI gets done.

**Comment by mitchute** [comment]
No point in waiting for the last windows build. Merging.

**Pull request merged**